### ISSUE TITLE
New-SPOTenantSite not working correctly

### ISSUE THREAD

**Issue by adrichema** [OfficeDev/PnP#1052]
When I try to create a new SharePoint Online site collection using the cmdlet New-SPOTenantSite the site will get created and on first sight everything looks okay, However, when I try to access a library the page shows the error "Cannot complete action" . I tried various kind of templates like STS#0 and BDR#0. 
When creating the site collection by hand, everything is working perfectly. SharePoint Online Powershell modules are also working as expected.

**Comment by erwinvanhunen** [comment]
This sounds more like there is an issue with your tenant, as the cmdlet itself is only forwarding the request to create the site collection to your tenant. Can you try creating a site collection using the user interface, do you have the same issue then?

**Comment by adrichema** [comment]
When creating the site collection by hand (using the user interface) everything is working perfectly. SharePoint Online Powershell modules are also working as expected. We had this problem on 2 separate development tenants.

**Comment by erwinvanhunen** [comment]
Can you try to create a new site collection using New-SPOSite (which has virtually the same parameter as the New-SPOTenantSite cmdlet)? New-SPOSite is part of the SharePoint Online PowerShell modules. Both New-SPOSite and New-SPOTenantSite use the same SDK / API behind the scenes to create the site collection.

**Comment by adrichema** [comment]
New-SPOSite is working fine as well

**Comment by erwinvanhunen** [comment]
Which version of the PnP Cmdlets do you have installed? It's this version that is the current version of Office 365: https://github.com/OfficeDev/PnP/blob/master/Binaries/PowerShell.Commands/PnPPowerShellCommands16.msi

There is a newer version available in the dev branch too: https://github.com/OfficeDev/PnP/blob/dev/Binaries/PowerShell.Commands/PnPPowerShellCommands16.msi

**Comment by adrichema** [comment]
I tried the master branch, now trying the dev branch....

**Comment by erwinvanhunen** [comment]
Just to be sure: are you connecting with Connect-SPOnline to your tenant admin site? e.g.

connect-sponline https://<tenant>-admin.sharepoint.com -etc. -etc.

**Comment by adrichema** [comment]
Same result with the dev branch ![error](https://cloud.githubusercontent.com/assets/13380526/8748726/c91285a8-2c9c-11e5-9b97-9e48b924e229.png)

**Comment by adrichema** [comment]
I did not connect to the tenant admin site, but I am not sure if this is causing the errors. Please look at the screenshot below.
![error1](https://cloud.githubusercontent.com/assets/13380526/8748828/60a971ce-2c9d-11e5-9f3d-77b481746a2a.png)

**Comment by erwinvanhunen** [comment]
As such is should not matter, but we're doing some context switching behind the scenes in order to create the site collection (the context used internally needs to be set up against the tenant admin site in order to create a site collection). This context switching does not occur when you connect directly with connect-sponline to the tenant admin site.

Regardless of that, it's a weird... we don't do anything specific other than in pseudo code:

'Hello Sharepoint Online, please create us a site collection at <this> url with <that> template'. We hand over the request then to SPO and the moment it's done we report back.

Another thing I just think of: when you execute the cmdlet, do you specify the -Wait parameter? If not, can you do that? Maybe you're accessing the site too early. Notice that when you execute the cmdlet without the Wait parameter it will create the site asynchronously, which means that the moment the prompt comes back the server is still creating the site. In your case it's maybe not done yet.

**Comment by adrichema** [comment]
Connecting to the tenant admin site did not make a difference. I did try waiting for more then 1 day, the site was still not working as expected.

**Comment by erwinvanhunen** [comment]
okay. Extremely puzzling...

Can you post the exact command or script you're using to create the site collection? So I can try to reproduce it here.

I will have a look at it tonight then.

**Comment by adrichema** [comment]
Thanks a lot for your support Erwin, I use the following command for creating the site.

$credentialsLabel = "mycredentials"
$gebruikersPortaalUrl = "https://xxxxxxx.sharepoint.com/sites/gebruikersportaal2"
$gebruikersPortaalTitle = "Gebruikersportaal"
$gebruikersPortaalDescription = "Hier komt een omschrijving voor het gebruikersportaal."
$gebruikersPortaalLanguageCode = 1043
$gebruikersPortaalTimezone = 110
$gebruikersPortaalTemplate = "BDR#0"
$gebruikersPortaalOwner =  "xxxxxx@xxxxxx.onmicrosoft.com"

Connect-SPOnline –Url https://xxxxxxxx-admin.sharepoint.com –Credentials $credentialsLabel
New-SPOTenantSite -Title $gebruikersPortaalTitle -Url $gebruikersPortaalUrl -Description $gebruikersPortaalDescription -Owner $gebruikersPortaalOwner -Lcid $gebruikersPortaalLanguageCode  -Template $gebruikersPortaalTemplate -TimeZone $gebruikersPortaalTimezone
Disconnect-SPOnline

**Comment by erwinvanhunen** [comment]
Okay, I tried and was able to reproduce the issue. It is due to the timezone you picked (110). 

Use Get-SPOTimeZone to pick a valid timezone value. When I navigated to the site settings in the site that had the issue, and changed in Regional Settings (in Dutch: Landinstellingen) the timezone to Amsterdam etc, (which is Timezone '4' when using the cmdlet) the Document library behaved as expected. 

For reference:

```
Get-SPOTimeZoneId -Match Amsterdam

Id Description                                 Identifier
-- -----------                                 ----------
 4 AMSTERDAM BERLIN BERN ROME STOCKHOLM VIENNA UTC+01:00
```
